src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -49,17 +51,25 @@
     }
 
     public static ArrayList<CountryNote> getSampleCountryNotes() {
-        return new ArrayList<>(Arrays.asList(
-            new CountryNote("This is a country note", new Country("SG")),
-            new CountryNote("This is another country note", new Country("MY"))
-        ));
-    }
+        CountryNote first = new CountryNote("Small domestic market as compared "
+                + "to rest of SEA countries", new Country("SG"));
+        Set<Tag> firstTags = new HashSet<>();
+        firstTags.add(new Tag("marketsize"));
+        first.setTags(firstTags);
+
+        CountryNote second = new CountryNote("Political unrest in the capital",
+                new Country("TH"));
+        Set<Tag> secondTags = new HashSet<>();
+        secondTags.add(new Tag("politics"));
+        second.setTags(secondTags);
+
+        CountryNote third = new CountryNote("A high-context country, hence building "
+                + "relationships is important for doing business", new Country("JP"));
+        Set<Tag> thirdTags = new HashSet<>();
+        thirdTags.add(new Tag("intercultural"));
+        third.setTags(thirdTags);
 
-    public static ArrayList<Note> getSampleClientNotes() {
-        return new ArrayList<>(Arrays.asList(
-            new Note("This is a regular note"),
-            new Note("This is another regular note")
-        ));
+        return new ArrayList<>(Arrays.asList(first, second, third));
```

Comment:
Should probably change this to return Arrays.asList and the return type to List<CountryNote>